mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulProfileScreen.kt

```diff
@@ -1,34 +1,40 @@
 package ch.epfl.sdp.mobile.state
 
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.remember
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
-import ch.epfl.sdp.mobile.application.Profile.Color
-import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
+import ch.epfl.sdp.mobile.application.Profile
 import ch.epfl.sdp.mobile.ui.profile.ProfileScreen
 import ch.epfl.sdp.mobile.ui.profile.ProfileScreenState
 import ch.epfl.sdp.mobile.ui.social.ChessMatch
 
-class AuthenticatedUserProfileScreenState(
-    private val user: AuthenticatedUser,
+class FetchedUserProfileScreenState(
+    user: Profile,
 ) : ProfileScreenState {
-  override val email = user.email
+  override val email = ""
```

Comment:
We should probably refine the hierarchy of `ProfileScreenState` so it doesn't receive an `email` (and does not need to display it).